Woof/Woof/Helper/AppStyle/String+IconName.swift

```diff
@@ -11,6 +11,8 @@ extension String {
         static let walkingsTab = "pawprint"
         /// The icon used for the "Profile" tab.
         static let profileTab = "person.text.rectangle"
+        /// The icon used for the "My schedule" tab.
+        static let scheduleTab = "calendar"
         /// The icon used for the 'Avatar' placeholder.
```

Comment:
nit:
```suggestion
        /// The icon used for the "Avatar" placeholder.
```

Reply:
It's not related to this PR, I didn't make these changes.